source/_integrations/fritz.markdown

```diff
@@ -29,77 +28,82 @@ ha_ssdp: true
+The AVM FRITZ!Box Tools integration allows you to control your [AVM FRITZ!Box](https://en.avm.de/products/fritzbox/) router and have presence detection for connected network devices.
 
 There is support for the following platform types within Home Assistant:
 
-- **Device tracker** - presence detection by looking at connected devices.
-- **Binary sensor** - connectivity status.
-- **Image** - QR code for Guest Wi-Fi.
-- **Button** - reboot, reconnect, firmware_update.
-- **Sensor** - external IP address, uptime and network monitors.
-- **Switch** - call deflection, port forward, parental control and Wi-Fi networks.
-- **Update** - firmware status of the device.
+- **{% term "Device tracker" %}** - presence detection by looking at connected devices.
+- **{% term "Binary sensor" %}** - connectivity status.
+- **{% term Image %}** - QR code for Guest Wi-Fi.
+- **{% term Button %}** - reboot, reconnect, firmware_update.
+- **{% term Sensor %}** - external IP address, uptime and network monitors.
+- **{% term Switch %}** - call deflection, port forward, parental control and Wi-Fi networks.
+- **{% term Update %}** - firmware status of the device.
+
 {% include integrations/config_flow.md %}
 
 {% important %}
-Both TR-064 and UPnP need to be enabled in the FRITZ!Box ( Home Network -> Network -> Network settings -> Access Settings in the Home Network ) for Home Assistant to login and read device info.
+Both the TR-064 (_Permit access for apps_) and UPnP (_Transmit status information over UPnP_) protocol needs to be enabled in the FRITZ!Box under **Home Network** > **Network** > **Network settings** > **Access Settings in the Home Network** for Home Assistant to login and read device info.
 {% endimportant %}
 
 ## Username
 
-The configuration in the UI asks for a username. Starting from FRITZ!OS 7.24 the FRITZ!Box creates a random username for the admin user if you didn't set one yourself. This can be found after logging into the FRITZ!Box and visiting System -> FRITZ!Box Users -> Users. The username starts with `fritz` followed by four random numbers. Under properties on the right it says `created automatically`. Prior to FRITZ!OS 7.24 the default username was `admin`.
+We strongly recommend to create a separate user to connect Home Assistant to your FRITZ!Box. This can be done under int the FRITZ!Box under **System** > **FRITZ!Box Users** > **Users** > **Add User**. The user needs the **FRITZ!Box Settings** permission.
+
+{% note %}
+If you still want to use the predefined user, please note that starting from FRITZ!OS 7.24 the FRITZ!Box creates a random username for the admin user if you didn't set one yourself. This can be found after logging into the FRITZ!Box and visiting **System** > **FRITZ!Box Users** > **Users**. The username starts with `fritz` followed by four random numbers. Under properties on the right it says `created automatically`. Prior to FRITZ!OS 7.24 the default username was `admin`.
```

Comment:
```suggestion
If you still want to use the predefined user, please note that as of FRITZ!OS 7.24, the FRITZ!Box creates a random username for the admin user if you didn't set one yourself. This can be found after logging into the FRITZ!Box and visiting **System** > **FRITZ!Box Users** > **Users**. The username starts with `fritz` followed by four random numbers. Under properties on the right it says `created automatically`. Prior to FRITZ!OS 7.24, the default username was `admin`.
```